calico-vpp-agent/policy/policy_server.go

```diff
@@ -84,7 +84,9 @@ type Server struct {
 	workloadsToHostPolicy *Policy
 	workloadAddresses     map[string]string
 	/* always allow traffic coming from host to the pods (for healthchecks and so on) */
-	allowFromHostPolicyId uint32
+	IngressAllowFromHostPolicyId uint32
+	EgressAllowFromHostPolicyId  uint32
```

Comment:
We can probably create a single `AllowFromHostPolicyId` containing both rules (RX & TX),
and assign it in Egress on the hep, and in Ingress on weps